src/gateway/kong-manager/auth/basic.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Enable Basic Auth for Kong Manager
+badge: enterprise
+---
+
+## Prerequisites
+
+To enable basic authentication, configure Kong with the following properties:
+
+```
+enforce_rbac = on
+admin_gui_auth = basic-auth
+admin_gui_session_conf = { "secret":"set-your-string-here" }
+```
```

Comment:
```suggestion
enforce_rbac = on
admin_gui_auth = basic-auth
admin_gui_session_conf = { "secret":"set-your-string-here" }
```
```

What should we do about this?

Reply:
Reworked this into a step. Let me know if that makes sense to you.